### PR TITLE
[IOPSC-59] Add endpoint to organizations services data export

### PR DESCRIPTION
**Pull request by LoFrance**
#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -340,6 +340,38 @@ if (config.IDP === "selfcare") {
       res.end();
     }
   );
+
+  app.use(
+    "/organizations/:organizationFiscalCode/services/*",
+    sessionTokenVerifier,
+    async (req, res) => {
+      const url = `${config.ORGANIZATION_SERVICES_URL}/organizations/${req.user?.organization.fiscal_code}/services${req.params[0]}`;
```
Comment (balanza) [inline src/app.ts:348]: What if `user` is null?

**Review by balanza — CHANGES_REQUESTED** [review]

File: src/app.ts (hunk 1 above)
Comment (balanza) [reply src/app.ts:348]: More: are you sure `user` always has a related `organization`? What if we are in a DevPortal installation (which would be the case  for admin users)?

Comment (LoFrance) [reply src/app.ts:348]: > What if `user` is null?

Organization Fiscal Code needs to be a value in the path "/organizations/:organizationFiscalCode/services/*", so I'm wondering if we can read the parameter in the URL path in this manner:

```
app.use(
    "/organizations/:organizationFiscalCode/services/*",
...
const organizationFiscalCode = req.params["organizationFiscalCode"];
const url = `${config.ORGANIZATION_SERVICES_URL}/organizations/${organizationFiscalCode}/services/${req.params[0]}`;
```

File: src/controllers/service_data.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import {
+  fromEither,
+  fromPredicate,
+  TaskEither,
+  tryCatch
+} from "fp-ts/lib/TaskEither";
+import nodeFetch from "node-fetch";
+
+// tslint:disable-next-line:no-any
+type ServiceDataSuccessResponse = any /* TODO: shape as the expected response from the API */;
```
Comment (balanza) [inline src/controllers/service_data.ts:33]: This should be the shape of the response expected from the API

Comment (LoFrance) [reply src/controllers/service_data.ts]: Fix with https://github.com/pagopa/io-developer-portal-backend/pull/184/commits/4d72bcc3efdc00892ffb6eff5bd96a0189066854

**Review by balanza — APPROVED** [review]

**Pull request merged**